admin_guide/diagnostics_tool.adoc

```diff
@@ -347,27 +347,19 @@ command, specify your cluster's inventory file and run the *_health.yml_*
 playbook:
 
 ----
-# ansible-playbook -i <inventory_file> \
-ifdef::openshift-enterprise[]
-    /usr/share/ansible/openshift-ansible/playbooks/openshift-checks/health.yml
-endif::[]
-ifdef::openshift-origin[]
-    ~/openshift-ansible/playbooks/openshift-checks/health.yml
-endif::[]
+# cd /usr/share/ansible/openshift-ansible
```

Comment:
I think we should make the instruction to cd into the folder that contains the Ansible playbooks a separate step or include a text direction to "change to the folder that contains the <playbooks>." It would better describe what we're doing and might help avoid confusion for users who are using the git flow

Do you have to be root to `cd` into this folder?

Reply:
I'm not sure the best way to approach the need to change directories. It's certainly a mess to have to put it everywhere and explain it everywhere but it's also pretty critical to do so.

No, you don't have to be root.

Reply:
If it's critical, then you have to say it each time, and it might need to be a warning instead of just something you mention offhand.

Then we should change those commands, too.